AFE/src/app/users/users-datasource.ts

```diff
@@ -0,0 +1,105 @@
+import { DataSource } from '@angular/cdk/collections';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { map } from 'rxjs/operators';
+import { Observable, of as observableOf, merge } from 'rxjs';
+
+// TODO: Replace this with your own data model type
+export interface UsersItem {
```

Comment:
It's ok, but we need to replace it in separate file
You could add it in: 
../users/shared/interfaces/interface.ts